stable/prometheus-cloudwatch-exporter/templates/deployment.yaml

```diff
@@ -53,12 +53,22 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /
+              path: /healthy/
```

Comment:
Shouldn't these be `/-/ready` and `/-/healthy` as per https://github.com/prometheus/cloudwatch_exporter/pull/116/files#diff-c526ccf413328707a9ea8def46d33f57R36? When I hit these paths, they just redirect me to `/`